swap/swap_test.go

```diff
@@ -1069,6 +1073,40 @@ func TestPeerProcessAndVerifyChequeInvalid(t *testing.T) {
 	}
 }
 
+func TestSwapLogToFile(t *testing.T) {
+
+	var err error
+
+	dir, err := ioutil.TempDir("", "swap_test_log")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(dir)
+
+	//Test log in fileSetSwaplogpath
+	debug.SetSwaplogpath(dir)
+
+	TestResetBalance(t)
+
+	files, err := ioutil.ReadDir(dir)
+	if err != nil || len(files) == 0 {
+		t.Fatal(err)
+	}
+
+	fileP := path.Join(dir, files[0].Name())
+
+	var b []byte
+	b, err = ioutil.ReadFile(fileP)
+	if err != nil {
+		t.Fatal(err)
+	}
+	s := string(b)
+	if !strings.Contains(s, "sending cheque") {
+		t.Fatalf("expected the log to contain sending cheque")
+	}
+
```

Comment:
please be mindful of unnecessary whitespace

Reply:
fixed